pkg_ci.es.Rmd

```diff
@@ -1,111 +1,111 @@
 
-We recommend using Codecov.
-To activate Codecov for your repo, run `usethis::use_github_action("test-coverage")` to create a file `.github/workflows/test-coverage.yaml`.
-You also need to give Codecov access to your github repository, see [Codecov quick start guide](https://docs.codecov.com/docs/quick-start) for how to set up access.
-Then add a Codecov status badge to the top of your README.md, see [Codecov status badges](https://docs.codecov.com/docs/status-badges).
+Para más detalles, consulta el [README del programa **covr** del paquete](https://github.com/r-lib/covr) para obtener instrucciones, así como en [`usethis::use_coverage()`](https://usethis.r-lib.org/reference/use_coverage.html) y [`usethis::use_github_action()`](https://usethis.r-lib.org/reference/github_actions.html).
 
-Currently, Codecov has access to all ropensci github repositories by default.
-When your repository is accepted and transferred to ropensci, Codecov access should transfer automatically.
-You will need to update the URL of the badge to point to the rOpenSci-hosted repository.
+Si ejecutas la cobertura en varios servicios CI [los resultados se fusionarán](https://docs.codecov.io/docs/merging-reports).
 
-For more details, see the [README for the **covr** package](https://github.com/r-lib/covr) for instructions, as well as [`usethis::use_coverage()`](https://usethis.r-lib.org/reference/use_coverage.html) and [`usethis::use_github_action()`](https://usethis.r-lib.org/reference/github_actions.html).
+## Más CI: OpenCPU
 
-If you run coverage on several CI services [the results will be merged](https://docs.codecov.io/docs/merging-reports).
+Después de la transferencia a la organización GitHub "ropensci" de rOpenSci, cada empuje al repo se construirá en OpenCPU y la persona que hace el commit recibirá un correo electrónico de notificación. Este es un servicio adicional de CI para los autores de paquetes que permite llamar a las funciones de R en los paquetes de forma remota a través de [https://ropensci.ocpu.io/](https://ropensci.ocpu.io/) utilizando la [API de opencpu](https://www.opencpu.org/api.html#api-json). Para más detalles sobre este servicio, consulta la página de ayuda de OpenCPU [página de ayuda](https://www.opencpu.org/help.html) que también indica dónde hacer preguntas.
 
-## Even more CI: OpenCPU
+## Aún más CI: rOpenSci docs {#rodocsci}
 
-After transfer to rOpenSci's "ropensci" GitHub organization, each push to the repo will be built on OpenCPU and the person committing will receive a notification email. This is an additional CI service for package authors that allows for R functions in packages to be called remotely via https://ropensci.ocpu.io/ using the [opencpu API](https://www.opencpu.org/api.html#api-json). For more details about this service, consult the OpenCPU [help page](https://www.opencpu.org/help.html) that also indicates where to ask questions.
```

Comment:
https://dev.ropensci.org/job/magick redirects me to https://ropensci.r-universe.dev/ui#packages. Are these URL still valid or is this documentation outdated?

Reply:
thank you!